DocSum/docker_compose/amd/gpu/rocm/compose.yaml

```diff
@@ -0,0 +1,89 @@
+# Copyright (C) 2024 Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  docsum-tgi-service:
+    image: ghcr.io/huggingface/text-generation-inference:2.3.1-rocm
+    container_name: docsum-tgi-service
+    ports:
+      - "${DOCSUM_TGI_SERVICE_PORT}:80"
+    environment:
+      no_proxy: ${no_proxy}
+      http_proxy: ${http_proxy}
+      https_proxy: ${https_proxy}
+      TGI_LLM_ENDPOINT: "http://${HOST_IP}:${DOCSUM_TGI_SERVICE_PORT}"
+      HUGGINGFACEHUB_API_TOKEN: ${DOCSUM_HUGGINGFACEHUB_API_TOKEN}
+    volumes:
+      - "/var/opea/docsum-service/data:/data"
+    shm_size: 1g
+    devices:
+      - /dev/kfd:/dev/kfd
+      - /dev/dri/${DOCSUM_CARD_ID}:/dev/dri/${DOCSUM_CARD_ID}
+      - /dev/dri/${DOCSUM_RENDER_ID}:/dev/dri/${DOCSUM_RENDER_ID}
+    cap_add:
+      - SYS_PTRACE
+    group_add:
+      - video
+    security_opt:
+      - seccomp:unconfined
+    ipc: host
+    command: --model-id ${DOCSUM_LLM_MODEL_ID}
+  docsum-llm-server:
+    image: ${REGISTRY:-opea}/llm-docsum-tgi:latest
```

Comment:
${REGISTRY:-opea}/llm-docsum-tgi:${TAG:-latest}, do not freeze the tag.

Reply:
revert tags, done